torch/csrc/jit/python/python_ivalue.h

```diff
@@ -49,8 +49,22 @@ struct C10_EXPORT ConcretePyObjectHolder final : PyObjectHolder {
     // when using C++. The reason is unclear.
     try {
       pybind11::gil_scoped_acquire ag;
```

Comment:
Shouldn't this one be part of `#else` flag? I.e. `gil_safe_call_once_and_store` sounds like a function that would acquire gil automatically before the call, wouldn't it?

Reply:
gil_safe_call_once_and_store is a type, not a function. So we need to hold the gil for this

Reply:
Also to clarify, even in 3.13t the gil can be enabled and thus needs to be properly handled.